roles/java/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Install JRE (Java Runtime Environment)
+  apt: name=default-jre state=present
+
+- name: Install JDK (Java Development Kit)
```

Comment:
We do already have [this](https://github.com/ZeusWPI/ansible-config/blob/master/roles/common/tasks/main.yml#L27) though, remove one of them